tutorials/tour/basics.md

```diff
@@ -0,0 +1,261 @@
+---
+layout: tutorial
+title: Basics
+
+disqus: true
+
+tutorial: scala-tour
+num: 2
+next-page: unified-types
+previous-page: tour-of-scala
+---
+
+In this page, we will cover basics of Scala.
+
+## Running Scala on Browser
+
+You can run Scala on browser by using [ScalaFiddle](https://scalafiddle.io).
+
+1. Go to https://scalafiddle.io.
+2. Copy and paste `println("Hello, world!")` to the left pane.
+3. Hit "Run" button and see it prints "Hello, world!" on the right pane.
+
+It is a perfect way for anybody to experiment a piece of Scala code anywhere, anytime!
+
+## Variables
+
+You can define variables with the `val` keyword.
```

Comment:
I'd side with @DarkDimius and @shogowada on this one; it's just friendlier to leave the sciencespeak out here. While "blah keyword binds an identifier to a computed expression" is the terminology you'd use in a PL fundamentals course or in a paper, I think most developers would find this terminology alienating. 

So yeah, how about this average between @shogowada's and @DarkDimius's suggestions:

"You can give names a computed expression using the `val` keyword. We call these _values_. Importantly, values are immutable; that is, once a computed expression is named with the `val` keyword, it cannot be reassigned to refer to a different computed expression."

(PS, we shouldn't use "immutable variable" as it's not exactly correct terminology)

WDYT?